src/pages/javascript/switch-statements/index.md

```diff
@@ -2,14 +2,71 @@
 title: Switch Statements
 ---
 ## Switch Statements
+<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+A `switch` statement in programming is similar to an `if-else` statement, but has the benefit of sometimes being easier to read when there are a lot of conditions, and it allows a `default` block, that will be executed if none of the other conditions are true.
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/javascript/switch-statements/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+#### Syntax:
+```
+switch(expression) {
+    case 1:
+        console.log('1');
+        break;
+    case 2:
+        console.log('2');
+        break;
+    default:
+        console.log('No true condition, default');
+}
+```
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+The above snippet shows the syntax for a basic `switch` statement. In this example, there are 3 different scenarios for:
+- `expression = 1`: First condition is true, and `1` gets printed to the console.
+- `expression = 2`: Second condition is true, and `2` gets printed to the console.
+- `expression = 'anything else'`: Case 1 and Case 2 are both false, so the default condition is executed.
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+The `default` condition is a condition that will be executed if none of the other cases are true.
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+##### Break
+The `break` keyword is required in each case to make sure that only the code in that case gets executed. Without the break, multiple cases could be executed. When JavaScript reaches a break keyword, it breaks out of the switch block. If the `break` was left out of the above example, this is what would happen:
```

Comment:
Add the code highlight for the break keyword here: "When JavaScript reaches a break keyword ..." -> "When JavaScript reaches a `break` keyword ..."

Reply:
Fixed, thanks!